python/n.py

```diff
@@ -148,16 +148,20 @@ def display_type(n_type):
 		return Fore.YELLOW + n_type + Style.RESET_ALL
 	elif isinstance(n_type, tuple):
 		return Fore.YELLOW + ' -> '.join(n_type) + Style.RESET_ALL
+	elif isinstance(n_type, list):
```

Comment:
Ah, you're using a list for tuples. Perhaps we could swap them so that tuples use tuples

Reply:
eee, maybe i am just more familiar with lists and can do things like `any([type(e) == lark.Token for e in expr])`